_data/projects/datapreprocessing.yml

```diff
@@ -0,0 +1,12 @@
+---
+name: datapreprocessing
+desc: Data preprocessing using Python
+site: https://github.com/TharinduDilshan/Data-Preprocessing
+tags:
+- machine learning
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/TharinduDilshan/Data-Preprocessing
```

Comment:
It'd be great if you had a curated list of issues for new contributors to pick up - is that something you're interested in?

Once you've got that list in place, we can update this to be the label assigned to those issues...